CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
     - Traffic Ops: Added an API 3.0 endpoint, /api/3.0/topologies, to create, read, update and delete flexible topologies.
     - Traffic Ops: Added support for `topology` query parameter to `GET /api/3.0/cachegroups` to return all cachegroups used in the given topology.
     - Traffic Portal: Added the ability to create, read, update and delete flexible topologies.
+- Added an API test for `DELETE /api/3.0/regions?name={{name}}`
```

Comment:
Should we be adding test changes to the changelog?

Reply:
This API test currently fails due to a regression in #4704. This PR will is intended to include a fix, and bugfixes should be included in the changelog. When the fix is pushed, I'll update the existing changelog entry instead of adding a new one.

Reply:
I don't think so. The audience for the changelog probably doesn't need to know about every single test added ¯\_(ツ)_/¯

Reply:
Whoops, your comment was added after I started writing mine @zrhoffman. But since this bug is unreleased (as of yet), I don't think the bugfix needs to be in the changelog either.

Reply:
Okay, changelog entry gone in da68d318a.